### PR TITLE
[W06H02] Fix pop test for MultiStack

### LINKED ISSUE #125: [W06H02] popShouldReturnCorrectValues tests irrelevant Stack contents
`popShouldReturnCorrectValues()` prüft den Stack auf Gleichheit mit dem Array `int[]{0}`, nachdem das letzte verbleibende Wert aus dem Stack ge`pop()`ed wurde:
https://github.com/MaximilianAnzinger/pgdp2223-tests/blob/f925297e3696b4a5226aad552e08ad429638044d/w06h02/test/pgdp/ds/MultiStackTest.java#L231
Laut Tutoren ist der Inhalt der ge`pop()`ed Zellen aber egal:
[https://zulip.in.tum.de/PGdP-W06H02/pop](https://zulip.in.tum.de/#narrow/stream/1480-PGdP-W06H02/topic/pop.28.29.20muss.20mem.20ver.C3.A4ndert.20werden.3F)
Stattdessen müsste geprüft werden, ob der `top`-Pointer gleich `-1` ist (und evtl, dass nach dem Ersten Stack keine weiteren mehr sind):
```java
int getTopPointer(Stack s) {
    try {
        final var field = s.getClass().getDeclaredField("top");
        field.setAccessible(true);
        return (int) field.get(s);
    catch (IllegalAccessException | NoSuchFieldException e) {
        throw new RuntimeException(e);
    }
}
```
und in `popShouldReturnCorrectValues()`:
```java
assertEquals(-1, getTopPointer(getHead()));
```

### PR DESCRIPTION
**Pull request by Vyquos**
Fixes #125.

### CODE REVIEW

**Review by HendrikHuebner — APPROVED** [review]
LGTM

**Comment by tdphucvn** [comment]
lgtm!

**Review by MaximilianAnzinger — APPROVED** [review]
[🐧](https://emojipedia.org/penguin/)

File: w06h02/test/pgdp/ds/MultiStackTest.java [hunk 1]
```diff
@@ -228,6 +238,7 @@ void popShouldReturnCorrectValues() {
         assertEquals(value4, stack.pop());
         assertEquals(value2, stack.pop());
 
-        assertArrayEquals(array(0), getMem(getHead()));
+        assertEquals(-1, getTopPointer(getHead()));
+        assertNull(getNext(getHead()));
```
Comment (nilsreichardt) [inline w06h02/test/pgdp/ds/MultiStackTest.java:242]: @felixhauptmann Is this correct? This is something that you added with #117

**Comment by MaximilianAnzinger** [comment]
Closed due to inactivity

**Pull request closed without merging**